lib/test.js

```diff
@@ -152,15 +152,15 @@ class Test {
 	}
 
 	countPassedAssertion() {
-		if (this.finishing) {
+		if (this.finished) {
 			this.saveFirstError(new Error('Assertion passed, but test has already ended'));
 		}
 
 		this.assertCount++;
 	}
 
 	addPendingAssertion(promise) {
-		if (this.finishing) {
+		if (this.finished) {
 			this.saveFirstError(new Error('Assertion passed, but test has already ended'));
```

Comment:
I considered that when I first wrote it, but the nuance seemed superfluous. And since #1330 it's not even shown anymore.